index.html

```diff
@@ -99,9 +99,9 @@
 	</div>
 
 	<div id="display-content">
-		<div class="section scrollspy" id="section_home">
+		<div class="section" id="section_home">
 			<a name="home"></a>
-			<div class="logos col-md-12 col-sm-12 col-lg-12">
```

Comment:
Instead of breaking things like this, I guess the desired result can be done by doing some tweaks in the css code instead for `#section_home` and `#section_about`.